### PR TITLE
Add Kosovo national team's flag

### PR DESCRIPTION
**Pull request by adamnfish**

### CODE REVIEW

File: source/crests/crests.json [hunk 1]
```diff
@@ -2009,6 +2009,11 @@
 		"team": "FYR Macedonia"
 	},
 	{
+		"paID": 64006,
+		"filename": "64006,.png",
+		"team": "Losovo"
```
Comment (gtrufitt) [inline source/crests/crests.json]: Losovo? 😆

File: source/crests/crests.json [hunk 2]
```diff
@@ -2009,6 +2009,11 @@
 		"team": "FYR Macedonia"
 	},
 	{
+		"paID": 64006,
+		"filename": "64006,.png",
```
Comment (gtrufitt) [inline source/crests/crests.json]: Also `,` in the filename?

**Comment by adamnfish** [comment]
#fail

**Comment by gtrufitt** [comment]
💯 😃

**Comment by adamnfish** [comment]
I'm not sure that `crests.json` file is actually used anywhere and I doubt it's been kept up to date, but included that just in case.

**Comment by jfsoul** [comment]
This need merging?

**Comment by adamnfish** [comment]
Yep, needs merging and someone on on the Frontend team to run the grunt script. That'll update the football assets the website uses and make the new flag available. I've chased @superfrank about it because he's done that in the past, but anyone on the team can do it.

**Comment by jfsoul** [comment]
ok thanks @adamnfish - I'll give it a go so we can close this

**Pull request merged**